### PR TITLE
Font sizes fixed and styling improvements

### LINKED ISSUE #740: Font size too small
**Description**
The font size for some text is too small. (See the last line in Getting Started or the text in the body in the following screenshots)

**Expected Behavior**
Increase the font size uniformly across the site.

**Screenshots**
Screenshot 2022-06-15 at 2 11 34 AM
Screenshot 2022-06-15 at 2 11 18 AM
Screenshot 2022-06-15 at 2 11 08 AM

**Enviroment:**
 - OS: [e.g. Ubuntu]
 - Browser: [e.g. Chrome, Safari]
 - Version: [e.g. 22]
 - Device: [e.g. laptop, iPhone 8]

---
[Optional] **To Reproduce**

[Optional] **Additional Context**

### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by eeshaanSA**
Signed-off-by: Eeshaan Sawant <eeshaans1@gmail.com>

**Description**
Fixed Font sizes at various places and did some styling improvements.
This PR fixes #740 

**Notes for Reviewers**
Always open for suggestions and improvements. Please point me out if anything should be added/corrected.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by debo19 — CHANGES_REQUESTED** [review]
The font size is small throughout the page, not just the getting started section.
Screenshot 2022-06-25 at 1 18 28 AM
Screenshot 2022-06-25 at 1 16 47 AM
We'll need a global fix for this.

**Comment by eeshaanSA** [comment]
Sure @debo19, than can be done. I thought the font part is fine for the rest of the part,(because it was even, one shape and size), except the Getting started, where the font was noticeably smaller. But if you want me to push a Global fix, sure it can be done. 
Will just increase the font size for all the texts that appear to be under the heading. Fine?

**Comment by eeshaanSA** [comment]
image

The image below is in a bigger font, same size which has been the largest for `<p> tags` so far. Do You want me to increase it more?

**Comment by eeshaanSA** [comment]
@debo19 /other maintainers please review the recent commit and let me know if there is anything else that is to be changed. 😊

**Review by warunicorn19 — COMMENTED** [review]
@debo19 don't you think the font size is a little too big now? 😅 or it might just be that I'm used to seeing the old size

**Comment by eeshaanSA** [comment]
@warunicorn19 even I think so, I can make it a little smaller, but then there wouldnt be a huge difference between the previous and the current version. 😅

File: _sass/catalog.scss
Comment (debo19) [inline _sass/catalog.scss:70]:
```suggestion
     font-size: 1.1rem;
```

Comment (debo19) [reply _sass/catalog.scss:70]: This should work

File: _includes/getting-started.html
Comment (debo19) [inline _includes/getting-started.html:3]: The style in the Getting Started heading was intentional. Can you change it back to the previous format?

Comment (eeshaanSA) [reply _includes/getting-started.html:3]: sure, no problem.

Comment (eeshaanSA) [reply _includes/getting-started.html]: done as requested. @debo19

File: _sass/catalog.scss
Comment (leecalcote) [reply _sass/catalog.scss:70]: I'll let @warunicorn19 and @debo19 make the final call here, though.

**Review by debo19 — COMMENTED** [review]
@eeshaanSA On the catalog pages, the heading looks smaller than the subtexts now. Maybe switch the `<h4>` with `<h3>`? Everything else looks good. 

Screenshot 2022-06-28 at 3 40 07 AM

**Comment by eeshaanSA** [comment]
can be done, @debo19, but if we change the heading to an h3, the main title, i.e (circuit breaker) will have to be an h2. Is there a workaround? 
Or should I just change the heading to an h3 and the main title to an h2?

**Comment by eeshaanSA** [comment]
image
It will look something on these lines if we do the changes I mentioned above. 
Need your opinions. @debo19 @warunicorn19 @leecalcote

**Comment by debo19** [comment]
Looks good @eeshaanSA

**Comment by eeshaanSA** [comment]
Sure @debo19. WIll do the required changes. But can you tell me where can I find the html file in order to make the required changes?
I tried modifying the catalog/technology/webassembly.html, but it didnt reflect the changes. Is there anything I am doing wrong?

**Comment by eeshaanSA** [comment]
image


Thanks for the confirmation. I recently noticed this issue is also seen on the respective pages of the patterns, where the heading looks kind of smaller than the paragraph. I will correct them as well, though I leave it up to you for the final call. @debo19 @warunicorn19

**Comment by eeshaanSA** [comment]
@debo @warunicorn19 request you to have a look.

**Comment by eeshaanSA** [comment]
@debo @warunicorn19 request you to have a look.

**Comment by debo** [comment]
@eeshaanSA please please please stop tagging me.

**Comment by eeshaanSA** [comment]
@debo19 please have a look.

**Review by debo19 — DISMISSED** [review]
LGTM 🎉 Thank you @eeshaanSA

**Comment by debo19** [comment]
@eeshaanSA will you address the merge conflict here, then this will be good to go

**Pull request merged**
